cg/store/crud/create.py

```diff
@@ -403,3 +406,14 @@ def add_sample_lane_sequencing_metrics(
             sample_internal_id=sample_internal_id,
             **kwargs,
         )
+
+    def add_order(self, order_data: Orderform):
```

Comment:
Feels a bit weird that we would have in the CRUD something which only works with orderforms? I would imagine we potentially want to expand this to a CLI method at some point where only the actual fields are provided? Could we change the parameters to data_analysis, customer_id and ticket_id or something similar?

Reply:
I hope we never create a cli command for placing an order 😭 But yes, the actual data is a `OrderIn` type which is reasonable to pass.

Reply:
I was thinking also for testing purposes, it is much easier to not mock a whole OrderForm. And still feels odd to me that the CRUD would "know" of anything about the indata. But up to you if you overrule me in this.

Reply:
I'll leave it as for now 👍